guides/v2.3/ui_comp_guide/components/ui-image-preview.md

```diff
@@ -0,0 +1,45 @@
+---
+group: ui-components-guide
+title: ImagePreview Component
+contributor_name: Shankar Konar
+contributor_link: https://github.com/konarshankar07
+---
+
+The ImagePreview component implements a detailed view image column. User can preview next or previous image by clicking next or previous arrow
+
+## Dependencies
+
+The ImagePreview component has dependencies on the following components:
+
+*  Column: `app/code/Magento/Ui/view/base/web/js/grid/columns/column.js`
+
+## Configuration options
+
```

Comment:
Global in this table: Add backticks to all Option names and Default Values, except null